src/main/java/org/gitlab4j/api/ContainerRegistryApi.java

```diff
@@ -0,0 +1,227 @@
+        });
+    }
+
+    /**
+     * Delete a registry repository tag.
+     *
+     * <pre><code>GitLab Endpoint: DELETE /projects/:id/registry/repositories/:repository_id/tags/:tag_name</code></pre>
+     *
+     * @param projectIdOrPath the project in the form of an Integer(ID), String(path), or Project instance
+     * @param repositoryId    the ID of registry repository
+     * @throws GitLabApiException if any exception occurs
+     */
+    public void deleteRepositoryTag(Object projectIdOrPath, Integer repositoryId, String tagName) throws GitLabApiException {
+
+        if (repositoryId == null) {
+            throw new RuntimeException("packageId cannot be null");
+        }
+
+        delete(Response.Status.NO_CONTENT, null, "projects", getProjectIdOrPath(projectIdOrPath), "registry", "repositories", repositoryId, "tags", tagName);
+    }
+
+    /**
+     * Delete repository tags in bulk based on given criteria.
+     *
+     * <pre><code>GitLab Endpoint: DELETE /projects/:id/registry/repositories/:repository_id/tags</code></pre>
+     * <p>
+     * This API call performs the following operations:
+     * <ol>
+     * <li>It orders all tags by creation date. The creation date is the time of the manifest creation,
+     * not the time of tag push.</li>
+     * <li>It removes only the tags matching the given name_regex.</li>
+     * <li>It never removes the tag named latest.</li>
+     * <li>It keeps N latest matching tags (if keep_n is specified).</li>
+     * <li>It only removes tags that are older than X amount of time (if older_than is specified).</li>
+     * <li>It schedules the asynchronous job to be executed in the background.</li>
+     * </ol>
+     * <p>
+     * These operations are executed asynchronously and it might take time to get executed. You can run this at most
+     * once an hour for a given container repository.
+     *
+     * @param projectIdOrPath the project in the form of an Integer(ID), String(path), or Project instance
+     * @param repositoryId    the ID of registry repository
+     * @param nameRegex       The regex of the name to delete. To delete all tags specify <code>.*</code>.
+     * @param keepN           The amount of latest tags of given name to keep.
+     * @param olderThan       Tags to delete that are older than the given time, written in human readable form
+     *                        <code>1h</code>, <code>1d</code>, <code>1month</code>.
+     * @throws GitLabApiException if any exception occurs
+     */
+    @SuppressWarnings({"RedundantThrows", "unused"})
+    public void deleteRepositoryTagBulk(Object projectIdOrPath, Integer repositoryId, String nameRegex, Integer keepN, String olderThan) throws GitLabApiException {
+        /*
+            This function is not implemented because we need the library to support FormData on DELETE requests.
+            See: https://docs.gitlab.com/ee/api/container_registry.html#delete-repository-tags-in-bulk
+         */
```

Comment:
It is highly unusual to have form data with the DELETE method.  Have you tried this with query params?

Reply:
I was just following the curl request set as example on the GitLab documentation. I will have a try with params.

Reply:
@PhilippeVienne 
Let me know how it goes.  If it requires form data, I'll add a ````delete()``` method that accepts form data.

Reply:
My local test have correctly a 202 Accepted response but the bulk DELETE seems to fail in all situation, as stated by documentation and also as you suggested with Query Params